## Figure legends

**Figure 1**.
Panicle complexity of 91 rice accessions.
The main component of variablity in panicle phenotypes splits accessions by domestication status, and is related to spikelet number, secondary branch number and primary branch number.
(**a**) We measured traits using spread panicles from *O. rufipogon*, *O. sativa*, *O. barthii* and *O. glaberrima*.
(**b**) The first principal component (PC1) in the panicle phenotype data accounts for 46.5% of variability and separates wild and domesticated accessions independently of continent.
(**c**) Spikelet number (SpN), secondary branch number (SBN) and primary branch number (PBN) have the highest loadings on PC1.
(**d**) Correlation between the main panicle traits that contribute to panicle architecture diversity.
Primary branch number and spikelet number correlate in wild species.
Secondary branch number and spikelet number correlate more in cultivated species than in wild species.
Primary and secondary branch numbers do not correlate.
RL: Rachis length;
PBL: Primary branch length;
PBIL: Primary branch internode length;
SBL: Secondary branch length;
SBIL: Secondary branch internode length; 
TBN: Tertiary branch number.

**Figure 2**.
Principal components analysis of transformed read counts for each library.
Principal component 5 (PC5) separates RNAseq samples by developmental stage, and explains 3.5% of total variability.
The first four components explain 86.3% of variability, and separate RNAseq samples by species.
Bars show single samples (three replicates per accession per stage).

**Figure 3**.
*AP2/EREBP*-like and MADS-box transcription factors change expression between IM and DM.
For each family, we plotted genes that were in the top 10% of all genes by absolute loading on PC5.
Genes in the upper panels had a positive loading on PC5 (corresponding to higher expression at the IM stage), whilst genes in the lower panels had a negative loading.
Most *AP2/EREBP*-like genes that pass the 10% cutoff were more highly expressed in the IM.
Three of the four *AP2/EREBP*-like genes that were more highly expressed in the DM belong to the AP2 clade.
*AP2/EREBP*-like genes that were more highly expressed in IM mainly belong to the ERF and DREB clades.
In contrast, most *MADS*-box genes that pass the cutoff were more highly expressed at the DM stage.
Clades for *AP2/EREBP*-like genes are from the Plant Transcription Factor Database v4.0 [@jinPlantTFDBCentralHub2017] and @sharoniGeneStructuresClassification2011.
*MADS*-box clades were manually tabulated from @aroraMADSboxGeneFamily2007.

**Figure 4**.
Mutants in two *AP2/EREBP*-like genes, *PLT8* and *ERF142*, have defects in panicle architecture compared to their background accessions.
The *crl5* mutant of *PLT8* (*LOC_Os07g03250*) produced fewer primary branches and spikelets, and the *smos1-3* mutant of *ERF142* (*LOC_Os05g32270*) produced fewer primary branches, secondary branches and spikelets.

**Figure 5**.
Gene expression clusters correlate with the main component of diversity of panicle architecture (PC1) and the number of secondary branches (SBN) and spikelets (SpN).
Clusters contained 19–31 genes each (Supporting information Table S8).
(**a**) Mean, scaled log~2~-fold change (L~2~FC) of genes by cluster and accession.
(**b**) Pearson correlation with PC1, SBN and SpN.
PC1 is the main principal component that separates panicles from domesticated and wild accessions of rice (Fig. 1).
Correlations with SBN and SpN are based on repeat panicle phenotyping for the accessions used for RNAseq in greenhouse conditions (Supporting information Fig. S8, Table S10).

**Figure 6**.
Parallel and divergent evolution of gene expression associated with domestication.
(**a**) Expression of genes with a stage × domestication interaction when all four accessions were tested together.
We used this test to identify genes where the change in L~2~FC between indeterminate (IM) and determinate (DM) stages changed in the same direction in both African and Asian domesticated accessions.
(**b**) Expression of genes with a stage × domestication interaction when tested separately for African and Asian accessions.
These genes have divergent changes between wild and domesticated accessions.
The genes plotted on panel **b** had an interaction in both of the separate tests, but not in the single test used to identify the genes in panel **a**.
