## Figure legends

**Figure 1 phenotype-pca**.
The main component of variablity in panicle phenotypes from 91 rice accessions splits accessions by domestication status, and is related to spikelet number, secondary branch number and primary branch number.
(**A**) We measured traits using spread panicles from *O. rufipogon*, *O. sativa*, *O. barthii* and *O. glaberrima*.
(**B**) The first principal component (PC1) in the panicle phenotype data accounts for 46.5% of variability and separates wild and domesticated accessions independently to continent.
(**C**) Spikelet number (SpN), secondary branch number (SBN) and primary branch number (PBN) have the highest loadings on PC1.
(**D**) Correlation between the main panicle traits that contribute to panicle architecture diversity.
Primary branch number and spikelet number correlate in wild species.
Secondary branch number and spikelet number correlate more in cultivated species than in wild species.
Primary and secondary branch numbers do not correlate.
RL: Rachis length; PBL: Primary branch length; PBIL: Primary branch internode length; SBL: Secondary branch length; SBIL: Secondary branch internode length; TBN: Tertiary branch number.

**Figure 2 transcriptome-pca**.
Principal components analysis of transformed read counts for each library.
Principal component 5 (PC5) separates RNAseq samples by developmental stage, and explains 3.5% of total variability.
The first four components explain 86.3% of variability, and separate RNAseq samples by species.
Bars show single samples (three replicates per accession per stage).

**Figure 3 MADS-AP2-heatmap**.
*AP2/EREBP*-like and MADS-box transcription factors change expression between IM and DM.
For each gene family, we plotted the 10% of genes that have the highest absolute loading on PC5.
Genes in the upper panels have a positive loading on PC5 (corresponding to higher expression at the IM stage), whilst genes in the lower panels had a negative loading.
Most *AP2/EREBP*-like genes that pass the 10% cutoff are more highly expressed in the IM.
Two of the four *AP2/EREBP*-like genes that are more highly expressed in the DM belong to the AP2 clade.
*AP2/EREBP*-like genes that are more highly expressed in IM mainly belong to the ERF clade.
In contrast, most MADS-box genes that pass the cutoff are more highly expressed at the DM stage.
Clades for *AP2/EREBP*-like genes are from the Plant Transcription Factor Database v4.0 [@jinPlantTFDBCentralHub2017].
MADS-box clades were manually tabulated from Figure 2 of @aroraMADSboxGeneFamily2007 (**Table S11 arora_clades**).

**Figure 4 panicle-mutants**.
Mutants in two *AP2/EREBP*-like genes, *PLT8* and *ERF142*, have defects in panicle architecture compared to their background accessions.
The *crl5* mutant of *PLT8* (*LOC_Os07g03250*) produces fewer primary branches and spikelets, and the *smos1-3* mutant of *ERF142* (*LOC_Os05g32270*) produces fewer primary branches, secondary branches and spikelets.

**Figure 5 cluster-phenotype-corr**.
Gene expression clusters correlate with the main component of divesity of panicle architecture (PC1), the number of secondary branches (SBN) and spikelets (SpN).
Clusters contained 19–31 genes each (**Table S8**).
(**A**) Mean, scaled log~2~-fold change (L~2~FC) of genes by cluster and accession.
(**B**) Pearson correlation with PC1, SBN and SpN.
PC1 is the main principal component that separates domesticated and wild accessions of rice (**Figure 1**).
Correlations with SBN and SpN are based on repeat panicle phenotyping for the accessions used for RNAseq in greenhouse conditions (Figure S10 **phenotyping-mpl**; Table S4 **PanicleTraitsPhenotypingPlantsSequenced**).]
The genes in clusters 1 and 2 may be related to domestication, because L~2~FC of genes in those clusters are higher or lower in domesticated accessions, respectively, and both clusters correlate with PC1 and SpN.
Clusters of genes with high L~2~FC in *O. sativa indica* have a positive correlation with SBN and SpN, whereas clusters of genes with low L~2~FC in *O. sativa indica* have a negative correlation with SBN and SpN.
Six of the 17 genes in Cluster 5, which has the highest L~2~FC in *O. sativa indica* and the highest correlation with SBN, are *AP2/EREBP*-like genes (*p*~adj~ = 7.6 × 10^-3^, hypergeometric test; Table S8 **clustered-genes**).
Conversely, nine of the 31 genes in Cluster 4, which has the lowest L~2~FC in *O. sativa indica* and the strongest negative correlation with SBN, are homeobox genes (*p*~adj~ = 2.17 x 10^-5^, hypergeometric test; Table S8 **clustered-genes**).

**Figure 6 dom-genes**.
Parallel and divergent evolution of gene expression during domestication.
(**A**) Expression of genes with a stage × domestication interaction when both domestications were considered have parallel changes in expression pattern in panicles at indeterminate (IM) and determinate (DM) meristem stage.
(**B**) Expression of genes with a stage × accession interaction in both domestications when tested separately, but not a stage × domestication interaction when both domestications were tested together.
These genes have divergent changes in expression between wild and domesticated accessions.
