## Figure legends

**Figure 1 phenotype-pca**. Panicle complexity of 93 rice accessions. **A**Spread mature rice panicule from *O. rufipogon*, *O. sativa*, *O. barthii* and *O. glaberrima*. We used spread panicles to measure panicle traits with P-TRAP [@al-tamPTRAPPanicleTrait2013]. The first principal component (PC1) in the panicle phenotye data accounts for 46.5% of variability and separates wild and domesticated accessions (**B**), and spikelet number (SpN), secondary branch number (SBN) and primary branch number (PBN) had the highest loadings on PC1 ( **F**). RL: Rachis length; PBL: Primary branch length; PBIntL: Primary branch internode length; SBL: Secondary branch length; SBIntL: Secondary branch internode length; TBN: Tertiary branch number.
>i update this figure..i put on the dropbox : Fig01propositionupdate.tiff

**Figure 2 transcriptome-pca**. Principal comonents analysis of transformed read counts for each library. Principal component 5 (PC5) separates RNAseq samples by developmental stage, and explains 5.4% of total variability. The first four components explain 51.7% of variability, and separate RNAseq samples by species.

**Figure 3 HB-AP2-heatmap**. AP2/EREBP and homeobox (HB) transcription factors change expression between BM and SM. **A**. AP2/EREBP and HB genes are distributed at the extremes of genes ranked on PC5 (GSEA permutation *t*-test: *p*~adj~ == 0.004 for both [**citation for GSEA**]). For the heatmap, we used the 10% of genes that have the highest absolute loading on PC5 (shown in red in the enrichment plot). **B**. Most AP2/EREBP genes that pass the cutoff are more highly expressed (**what's on the scale bar?**) in the BM. Three of the four AP2/EREBP genes that are more highly expressed in the SM belong to the AP2 subfamily. Genes that are more highly expressed in BM mainly belong to RAV, DREB and ERF subfamilies. **C**. Most HB genes that pass the cutoff are more highly expressed in the SM. Ten out of twenty of these genes belong to the HD−ZIP IV subfamily (**are there stats for this?**).

**Figure 4 cluster-phenotype-corr**. Genes with a high log~2~-fold change (L~2~FC) in *O. sativa indica* are correlated with increased production of primary branches (PBN), secondary branches (SBN) and spikelets (SpN).

**Figure 5 panicle-mutants**. Mutants in three AP2/EREBP-like genes, *PLT8*, *ERF142* and *DLT*, have defects in panicle architecture compared to their background accessions. The *PLT8* mutant produces fewer primary branches and spikelets. The mutants of *ERF142* and *DLT* both produce fewer primary branches, secondary branches and spikelets.


## Supplementary figures

**Figure S1 correlation-pbn-spn**.
Correlation between the main panicle traits that contribute to panicle architecture diversity.
Primary branch number and spikelet number correlate in wild species. Secondary branch number and spikelet number correlate more in cultivated species than in wild species.
Primary and secondary branch numbers don't correlate, suggesting that they
are controlled by different genetic mechanisms.
> could you add the correlation value on each panel

**Figure S2 suppl-phenotype-pca-all-pc**. Principal component analysis (PCA) of panicle phenotyping data showing components 1–4. PC1 accounts for 46.5% of variability and separates panicles from domesticated and wild accessions. The lower ordinates do not separate panicles by species.

**Figure S3 suppl-phenotype-all-varieties**. The accessions used for RNAseq are consistent with species-wide patterns of panicle architecture. Scores on PC1 for the accessions chosen for RNAseq are shown in red. (**Describe what the boxplot shows. How many points are there for each accession?**)

**Figure S4 qpcr-confirms-sampling**. **A** Developmental stages of immature panicles collected for expression analysis. Stage 1  : rachis meristem; Stage2 : formation of primary branch meristems, elongation of primary branch meristem and formation of axillary meristem; Stage3, spikelet meristem  and floret differentitaion; Stage 4, floral organ differenciation/development. M, axillary merisyem, Fl, flower; Sp, spikelet, RM, Rachis meristem; PbM, primary branch meristem; ePbM, primary branch elongated; Flm, floret meristem; St, stamen; p, palea; l, lemma. **B** Quantitative RT-PCR using meristem stage specific marker genes for validation of all stages among species. OsMADS1/LHS1 is used as a markers of Spikelet meristem differentiation. FZP is used as... LAX1....TAW .... osMADS14.....
> Relabel the x-axis to match stage abbreviations from results

**Figure S5 phenotyping-mpl**. Detailed phenotyping of five Oryza accessions. The three domesticated accessions produce more spikelets than their wild relatives. In comparison to *O. sativa japonica*, *O sativa indica* produces more secondary branches.

**Figure S6 distance-heatmap**. Heatmap of pairwise distances between libraries.

**Figure S7 NAC-MADS-SPL-heatmap**. MADS and SBP genes are more highly expressed in the SM. NAC genes are more highly expressed in the BM.

**Figure S8 cluster-5-details**. Most genes in cluster 5 have negative L~2~FCs between BM and SM in *O. rufipogon*, *O. barthii* and *O. glaberrima*, but L~2~FCs in *O. sativa indica* that are of smaller magnitude and closer to zero. This cluster has an enrichment of AP2/EREBP genes.

**Figure S9 lmd-paper-ap2**. Expression of AP2/EREBP-like genes in *O. sativa japonica* cv. Nipponbare meristems. Data from [@harropGeneExpressionProfiling2016].

## Supplementary tables

**Table S1 Plantinfo**. List of rice accessions phenotyped for panicle diversity

**Table S2 supp-table-PrimerList**. List and sequenced if pirmers used.

**Table S3 PanicleTraitsPhenotyping**. Panicle phenotyping data from `P-TRAP` analysis measured in the 93 accessions from wild and domesticated Asian and African rice species. 

**Table S4 PanicleTraitsPhenotypingPlantsSequenced**. Panicle traits data from P-TRAP analysis of rice accessions used for sequencing analysis.

**Table S5 mapping-statistics**. Read and mapping statistics for all RNAseq libraries.

**Table S6 DE-genes-stages**. Genes diffentially expressed between stages across all species.

**Table S7 DE-genes-interaction**. stage × accession DE genes

 
