## Results

To reveal gene expression patterns associated with diversity of inflorescence architecture, we used detailed phenotyping of panicles from domesticated and wild accessions of Asian and African rice, followed by whole-transcriptome sequencing (RNASeq) [and analysis of single-gene mutants with defects in panicle branching]. **expand slightly**

### Panicles from domesticated accessions produce more primary branches, secondary branches and spikelets

To measure the diversity of panicle architecture, we phenotyped **n** accessions of wild Asian rice (*Oryza rufipogon*; figure 1A), domesticated Asian rice (*Oryza sativa*; figure 1B), wild African rice (*Oryza barthii*; figure 1C) and domesticated African rice (*Oryza glaberrima*; figure 1D), using `P-TRAP` for automated measurement of traits [@al-tamPTRAPPanicleTrait2013]. The first principal component in the phenotyping data accounts for 46.5% of variability, and separates domesticated and wild accessions but not Asian and African accessions (figure 1E). Spikelet number, secondary branch number and primary branch number have the highest loadings on PC1, suggesting that these factors are the main differences between panicles from wild and domesticated accessions. Lower-order components do not separate panicles from different accessions (figure S1). **Correlations**.

### Differences in gene expression profiles between wild and domesticated panicles

To investigate gene expression associated with variation in panicle architecture, we chose a single accession for each of *O. rufipogon*, *O. sativa* japonica (Nipponbare), *O. sativa* indica (IR84), *O. barthii* (**?**) and *O. glaberrima* (**TOG...**). We performed further phenotyping on these accessions to ensure that they were consistent with species-wide patterns of panicle architecture (Figure S3).

## Figure legends

**Figure 1**. Panicles from domesticated accessions produce more branches and spikelets than panicles from wild accessions. We used spread panicles from **A** *O. rufipogon*, **B** *O. sativa*, **C** *O. barthii* and **D** *O. glaberrima* to measure panicle phenotypes with P-TRAP [@al-tamPTRAPPanicleTrait2013]. The first principal component (PC1) in the panicle phenotye data accounts for 46.5% of variability and separates wild and domesticated accessions (**E**), and spikelet number (SpN), secondary branch number (SBN) and primary branch number (PBN) had the highest loadings on PC1 ( **F**). RL: Rachis length; PBL: Primary branch length; PBIL: Primary branch internode length; SBL: Secondary branch length; SBIL: Secondary branch internode length; TBN: Tertiary branch number; PL: Panicle length.

**Figure S1**. Principal component analysis (PCA) of panicle phenotyping data showing components 1â€“4. PC1 accounts for 46.5% of variability and separates panicles from domesticated and wild accessions. The lower ordinates do not separate panicles by species.

**Figure S2** Correlation between the main traits that determine panicle phenotype. A. Primary Branch Number and Spikelet Number correlates only slightly mainly in wild species. B. Secondary Branch Number and Spikelet number
highly correlates in cultivated species, less in wild species. C.
Primary and Secondary Branches poorly correlates, suggesting that they
are controlled by different genetic mechanisms