## Supporting information

**Figure S1**.
Spread mature rice panicle.

**Figure S2**.
Principal component analysis (PCA) of panicle phenotyping data showing components 1–4.

**Figure S3**.
The accessions used for RNAseq are consistent with species-wide patterns of panicle architecture.

**Figure S4**.
Early stages of rice panicle development used for gene expression analysis.

**Figure S5**.
Heatmap of pairwise distances between RNAseq samples.

**Figure S6**.
Expression of genes from selected transcription factor families.

> To remove

**Figure S7**.
Expression of *AP2/EREBP*-like genes in *O. sativa japonica* cv. Nipponbare meristems [data from @harropGeneExpressionProfiling2016].

**Figure S8**.
Phenotyping of the five *Oryza* accessions used for RNAseq.

**Figure S9**.
Most genes in cluster 5 have negative L~2~FCs between IM and DM in *O. rufipogon*, *O. barthii* and *O. glaberrima*, but L~2~FCs in *O. sativa indica* are closer to zero.

**Figure S10**.
Expression analysis along early panicle development of *AP2/EREBP*-like genes present in cluster 5.

**Table S1**.
Rice accessions used in this study.

**Table S2**.
Sequences of primers used.

**Table S3**.
Quantification of panicle traits in 91 accessions from wild and domesticated Asian and African rice species.

**Table S4**.
Read and mapping statistics for all RNAseq samples.

**Table S5**.
Transcription factor family enrichment by L~2~FC.

**Table S6**.
Differential expression test results between stages across all species.

**Table S7**.
Quantification of panicle traits from crl5 and smos1 mutants.

**Table S8**.
Clustered genes.

**Table S9**.
Differential expression test results for the stage × accession interaction in Asian and African accessions.

**Table S10**.
Detailed quantification of panicle traits from rice accessions used for sequencing analysis.
