### Supporting information table captions

**Table S1**.
Rice accessions used in this study.

**Table S2**.
Sequences of primers used.

**Table S3**.
Quantification of panicle traits in 91 accessions from wild and domesticated Asian and African rice species.
Ob, *O. barthii*;
Og, *O. glaberrima*;
Os, *O. sativa*;
Or, *O. rufipogon*;
PBN,Primary branch;
PBIL, Primary branch internode length;
PBL, Primary branch length;
RL, Rachis length;
SBN, Secondary branch;
SBIL, Secondary branch internode length;
SBL,Secondary branch length;
SpN, Spikelet number;
TBN, Tertiary branch number.

**Table S4**.
Read and mapping statistics for all RNAseq samples.

**Table S5**.
Transcription factor families that are enriched along PC5.

**Table S6**.
Differential expression test results between stages across all species.

**Table S7**.
Detailed quantification of panicle traits from *crl5* and *smos1* mutants.
PBN,Primary branch;
PBIL, Primary branch internode length;
PBL, Primary branch length;
RL, Rachis length;
SBN, Secondary branch;
SBIL, Secondary branch internode length;
SBL,Secondary branch length;
SpN, Spikelet number;
TBN, Tertiary branch number

**Table S8**.
Clustered genes.

**Table S9**.
Differential expression test results for the stage Ã— accession interaction in Asian and African accessions.

**Table S10**.
Detailed quantification of panicle traits from rice accessions used for sequencing analysis.
These plants were grown at the same time and in the same conditions as the plants used for gene expression analysis. 
